opencti-platform/opencti-graphql/src/modules/deleteOperation/deleteOperation-domain.ts

```diff
@@ -117,8 +117,11 @@ const resolveEntitiesToRestore = async (context: AuthContext, user: AuthUser, de
       const missingIds = [...targetIdsToFind].filter((id) => !availableIds.includes(id));
       const targetsFromTrash = await elFindByIds(context, user, missingIds, { baseFields: ['internal_id', 'entity_type'], indices: READ_INDEX_DELETED_OBJECTS }) as BasicStoreObject[];
       if (targetsFromTrash.length > 0) {
-        const name = extractRepresentative(targetsFromTrash[0]); // only hint the first one
-        throw FunctionalError(`Cannot restore: a relationship targets a deleted element [${name.main}], restore it before retrying`);
+        // only hint the first 3 ones
+        let name = extractRepresentative(targetsFromTrash[0]).main;
+        if (targetsFromTrash.length > 1) name = `${name},${extractRepresentative(targetsFromTrash[1]).main}`;
+        if (targetsFromTrash.length > 2) name = `${name},${extractRepresentative(targetsFromTrash[2]).main}`;
```

Comment:
I think this can be simplified like this : 

```
let name = targetsFromTrash.slice(0, 3).map((t) => extractRepresentative(t).main).join(',');
if (targetsFromTrash.length > 3) name = `${name}, ... and ${targetsFromTrash.length - 3} more`;
```
With a suggestion to display the number of other entities with ', ... and X more'